README.md

```diff
@@ -40,21 +40,28 @@ It is perfectly okay to simply note down your thoughts for this, and not write a
 If you *do* implement a solution, you can test it using the product IDs provided in the file `product_feed.py`, or find other product IDs from Coolshop. Note that any product might at any point in time become unavailable.
 Product data can be downloaded from `https://www.coolshop.dk/api/products/{id}`.
 
-## D) Premium and Date Handling
-We provide a Premium subscription, to let customers buy products cheaper.
-The first time a customer signs up for Premium, they get an entire month for free, after that, each month costs 39 DKK.
-Payment happens the same day each month, e.g. if a Premium has signed up the 15th, the money will be charged the 15th every month.
-To prevent February from causing problems, subscriptions created on the 29th, 30th, and 31st of any month, behave as if they were created on the 1st of the following month instead.
-When a Premium subscription is cancelled, the customer is still a Premium member for the rest of the payment period.
-If the cancellation happens on the payment day, it is assumed the money has already been charged,
+## D) Calculate next delivery day
+Customers often like to know when their orders are delivered, so we want to calculate an expected delivery date.
 
-* Implement the function `get_first_payment_date(signup_date: date, previous_premium: boolean) -> date`.
-* Implement the function `get_end_date(sign_up_date: date, cancel_date: date) -> date`.
-* Implement the function `get_billing_day(signup_date: date) -> int`.
+We deliver orders within 1-2 work days in Denmark. If the order is placed before 15:00 on a work day the customer can expect the package the following work day. If the order is placed on a non work day or after 15:00 it will be delivered after 2 work days.
```

Comment:
I _think_ the more correct term is working days, not entirely sure though 🤷🏻‍♂️ 

```suggestion
We deliver orders within 1-2 working days in Denmark. If the order is placed before 15:00 on a working day the customer can expect the package the following working day. If the order is placed on a non work day or after 15:00 it will be delivered after 2 working days.
```